tests/test_concurrent_submissions.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+# © 2016-2021 Resurface Labs Inc.
+
+import random
+import time
+from typing import List
+
+from tests.test_helper import *
+from usagelogger import BaseLogger
+
+
+def test_background_workers() -> None:
+    logger = BaseLogger(MOCK_AGENT, url=DEMO_URL)
+    for _ in range(10):
+        logger.submit("{}")
+        time.sleep(random.random() * 2)
+    logger.wait_for_response()
+    assert logger.submit_successes == 10
+
+
+def test_multiple_loggers() -> None:
```

Comment:
Since we are checking multiple threads maybe we can assert the activate threads to make sure that multi-thread is running.